### DISCUSSION TITLE
Pdf (v3.1) compiling error

### DISCUSSION THREAD

**Discussion by GIGASHAB** [joaomlourenco/novathesis#267] [Q&A]
Hi,

I´ve just started to write my dissertation in the overleaf. It is an error while compiling the pdf:
"
This compile didn’t produce a PDF. This can happen if:
There is an unrecoverable LaTeX error. If there are LaTeX errors shown below or in the raw logs, please try to fix them and compile again.
The document environment contains no content. If it’s empty, please add some content and compile again.
This project contains a file called output.pdf. If that file exists, please rename it and compile again.
"

**Comment by joaomlourenco** [comment]
Please report which version of the template are you using?  How di you start it in overleaf?  
From the Overleaf templates? Which link? 
Did you change something before compiling? 
 Is there a file named "output.pdf"????

**Comment by GIGASHAB** [comment]
Version 3.1 ,
For starting in Overleaf, I uploaded the zip file provided in the Git ,
Not in the commands, Just added texts in the body( providing contents),

**Comment by joaomlourenco** [comment]
Please follow the instructions in the template web page for starting LaTeX in Overleaf. 

Also, version 3.1 is VERY OLD.

I advise you to go to google and search for "novathesis overleaf" and then "open as a project".
